tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -1804,20 +1804,28 @@ public function checkIfShareExists(string $share, string $sharee, string $sharer
 				$foundShareInSharedWithMe = true;
 			}
 		}
-		Assert::assertTrue($foundShareInSharedWithMe, "Share '$share' was not found in the shared-with-me list");
+
+		if ($shouldOrNot === "should not") {
+			Assert::assertFalse($foundShareMountpoint, "Share mountpoint '$share' was found in the drives list");
```

Comment:
this message is for all error msg below so please look in other lines too, better if error message says what expected and  actual value found

Reply:
IMHO,
these assertions will be helpful for specific expected and actual. Because `$share` is what came from the user input. Isn't it?

Reply:
Like this, it tells what was expected and what was found .
```suggestion
			Assert::assertFalse($foundShareMountpoint, "Expected share to be empty but found share '$share' ");
```